### PR TITLE
[Docs] Better PR template, repo cleanup

### PR DESCRIPTION
**Pull request by johnsaigle**
### Brief summary of changes

The current PR template isn't perfect, so this is an approach at reworking it again. Instructions are hidden in HTML comments so do not need to be manually deleted by developers. This also allows for much more detailed instructions.

This PR also moves the CONTRIBUTING and PULL_REQUEST_TEMPLATE github files to the [.github/ folder](https://blog.github.com/2016-02-17-issue-and-pull-request-templates/) to keep things clean.

### CODE REVIEW

**Comment by driusan** [comment]
HTML comments just prevent it from showing up on GitHub, they still need to be removed or they'll pollute the git history.

I think I'm the only one who doesn't like the template, you can just move it to the .github directory as-is (maybe remove the last sentence which some other people seem to have a problem with based on some other passive-aggressive PR descriptions that I've seen.)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!--
+Thanks for contributing to LORIS!
+You can help us keep our project organized and efficient by filling out
+the information below.
+
+After you have provided detailed information here, please apply appropriate 
+labels (and milestones, if you know them). This will help us to review and test
```
Comment (ridz1208) [inline .github/PULL_REQUEST_TEMPLATE.md]: please remove milstones, they are set after merge of the PR

**Comment by Jkat** [comment]
Not a big deal but was wondering: should `### Description` be commented too? Since the text area _is_ already the description of the PR itself

**Comment by driusan** [comment]
@johnsaigle ?

**Comment by johnsaigle** [comment]
@driusan I did something weird with git and also can't reopen this. Gonna just issue a new PR where I move the templates as-is like you suggested.

**Pull request closed without merging**